components/Callout/style.scss

```diff
@@ -7,6 +7,7 @@
     border-color: var(--border);
     color: var(--text);
     padding: $l-offset;
+    min-height: 3 * $l-offset;
```

Comment:
Hmmm, interesting that we need to set a minimum height at all here… (And also kinda strange to see this left offset var being used to set the height!)

Reply:
Probably not a huge deal, but it does suggest that there was some underlying change to the callout markup? It'd be nice to know what that is/why it broke the layout!

Reply:
This is/was actually a pre-existing condition? Perhaps it was a regression I introduced, but it is happening in core `@readme/markdown`.

Reply:
Ahhh, I see, I see… Unlikely it's some you did though! I just probably never tested this against my CSS since an entirely empty callout is (technically) a case we don't support.

Reply:
Interesting—even GitHub doesn't handle empty quotes particularly gracefully… Check it out:

> 

Honestly, thinking about it a little harder now, maybe we can just remove this line entirely?

```suggestion
```

Reply:
A+